app/javascript/components/widgets/forest-change/tree-loss-tsc/index.js

```diff
@@ -57,8 +57,8 @@ export default {
     },
     // loss tsc
     {
-      dataset: '89755b9f-df05-4e22-a9bc-05217c8eafc8',
-      layers: ['fd05bc2c-6ade-408c-862e-7318557dd4fc']
+      dataset: 'f23a1803-ba17-4fe3-a1ad-39abfbfb56c6',
```

Comment:
hey, now all the IDs of the datasets and layers that are used somewhere in the codebase are moved to `layers-datasets.js` and `layers.js` files, could you move it to consts there? 🙏

Reply:
ah great. will do

Reply:
Hmm this means it wont work on staging as they will have different IDs.

Reply:
we don't have dedicated datasets and layers for staging and production, these are the layers that we have: https://docs.google.com/spreadsheets/d/1V0tyqGoqOIfzMwx8f0Ix0-KExM1I63wCgUgbPoCW708/edit#gid=148980076

Reply:
Ok good to know. Updated.